pipenv/core.py

```diff
@@ -1158,6 +1158,13 @@ def do_purge(bare=False, downloads=False, allow_global=False):
         click.echo(
             fix_utf8("Found {0} installed package(s), purging…".format(len(to_remove)))
         )
+    
+    # Skip purging if there is no packages which needs to be removed
+    if not to_remove:
+        if not bare:
+            click.echo(crayons.green("Environment now purged and fresh!"))
```

Comment:
I'd prefer to be a bit more explicit about this and say exactly what happened, i.e. that there was nothing to remove.

Then we should return success

Reply:
Yes, that will be better. How about now?

Now the result looks like followings.

```
Un-installing all packages from virtualenv...
Found 0 installed package, skip purging
Environment now purged and fresh!
```

Reply:
Perfect